src/application/health_checks/health_checks.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * health_checks.h
+ *
+ * Created on: Nov 24, 2024
+ *
+ */
+
+#ifndef HEALTH_CHECKS_H_
+#define HEALTH_CHECKS_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#define ADC1_MAX_COUNTS 65535.0f
+#define ADC1_VREF 3.3f
+#define R_SENSE 0.033f
+#define INA180A3_GAIN 100.0f
+
+static inline float adc1_voltage_v(uint32_t counts) {
+    return (((float)(counts)) / ADC1_MAX_COUNTS) * ADC1_VREF;
+}
+
+static inline float adc1_curr_ma(float voltage) {
+    return (voltage / INA180A3_GAIN / R_SENSE);
```

Comment:
You may end up with a division by 0 error in edge cases

Reply:
Since gain and r_sense are defined constants, how would there be a division by 0?

Reply:
nvm I thought these may be variables